codalab/client/json_api_client.py

```diff
@@ -678,3 +678,10 @@ def update_worksheet_raw(self, worksheet_id, lines):
             path='/worksheets/%s/raw' % worksheet_id,
             headers={'Content-Type': 'text/plain'},
             data='\n'.join(lines))
+
+    @wrap_exception('Unable to fetch worker information')
+    def get_worker_info(self):
```

Comment:
rename to `get_workers_info` (plural); also applies to stuff in `bundle_cli.py`.